diff_cover/violationsreporters/java_violations_reporter.py

```diff
@@ -115,3 +116,46 @@ def installed(self):
         Returns: boolean False: As findbugs analyses bytecode, it would be hard to run it from outside the build framework.
         """
         return False
+
+
+class PmdXmlDriver(QualityDriver):
+    def __init__(self):
+        """
+        See super for args
+        """
+        super(PmdXmlDriver, self).__init__(
+            'pmd',
+            ['java'],
+            []
+        )
+
+    def parse_reports(self, reports):
+        """
+        Args:
+            reports: list[str] - output from the report
+        Return:
+            A dict[Str:Violation]
+            Violation is a simple named tuple Defined above
+        """
+        violations_dict = defaultdict(list)
+        for report in reports:
+            xml_document = cElementTree.fromstring("".join(report))
+            files = xml_document.findall(".//file")
+            for file in files:
```

Comment:
Please rename the `file` variable here. 

File is a type in python so using it as a variable name can be confusing.

```
>>> file
<type 'file'>
```